raiden/transfer/state_change.py

```diff
@@ -223,6 +224,42 @@ def from_dict(cls, data):
         )
 
 
+class ActionChannelSetFee(StateChange):
+    def __init__(self, canonical_identifier: CanonicalIdentifier, mediation_fee: FeeAmount):
+        self.canonical_identifier = canonical_identifier
+        self.mediation_fee = mediation_fee
+
+    def __repr__(self):
+        return f'<ActionChannelSetFee id:{self.canonical_identifier} fee:{self.mediation_fee}>'
```

Comment:
question: Canonical identifier is a named tuple. So this will appear kind of funny when shown as a string. Shouldn't you maybe add a `__str__` function to the Canonical identifier?

Reply:
Done